build.sbt

```diff
@@ -140,7 +140,8 @@ lazy val transactionImpl = (project in file("transaction-impl"))
     libraryDependencies ++= Seq(
       lagomJavadslPersistenceCassandra,
       lagomJavadslTestKit,
-      lagomJavadslKafkaBroker
+      lagomJavadslKafkaBroker,
+      "com.datastax.cassandra" % "cassandra-driver-extras" % "3.0.0"
```

Comment:
We can probably extract this into a variable. We're using it in few different places now.